test/k8sT/Services.go

```diff
@@ -2169,9 +2169,17 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 						var ccnpHostPolicy string
 
 						BeforeAll(func() {
-							DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, map[string]string{
+							options := map[string]string{
 								"hostFirewall": "true",
-							})
+							}
+							// We can't rely on gke.enabled because it enables
+							// per-endpoint routes which are incompatible with
+							// the host firewall.
+							if helpers.RunsOnGKE() {
+								options["gke.enabled"] = "false"
+								options["tunnel"] = "disabled"
+							}
```

Comment:
I wonder if this is worth factoring as a helper? Probably not for now since it occurs only two times, but in case it happens at more locations in the future? :thinking:

Reply:
This won't be needed once we have support for per-endpoint routes in GKE. I'm planning to send the PR for that support this week.